sdk/python/requirements.txt

```diff
@@ -4,4 +4,3 @@ six >= 1.10
 python_dateutil >= 2.5.3
 setuptools >= 21.0.0
 urllib3 >= 1.15.1
-kubernetes
```

Comment:
Did you run post_gen.py? it should add `"kubernetes"` to the requirements.txt file.

Reply:
I thought you said it runs when you run gen.py. 

Reading that script I think it adds kubernetes to test-requirements.txt but not to requirements.txt.

Reply:
Ah right it is only needed for the unit tests... Nevermind :)

Reply:
That’s not exactly true.  I had to add this dependency to pyproject.tomo to import it as a library.  The unit tests pass because we include test dependencies and those work.

Reply:
I noticed we import kubernetes here https://github.com/kubernetes-sigs/jobset/blob/main/sdk/python/jobset/models/__init__.py